sonaranalyzer-dotnet/its/expected/akka.net/Akka.Tests.Shared.Internals-{E636D23C-3432-4AA9-9A5D-5F29D33D3399}-S100.json

```diff
@@ -15,7 +15,7 @@
 },
 {
 "id":  "S100",
-"message":  "Rename method 'expectMsgPF' to match camel case naming rules, consider using 'ExpectMsgPF'.",
+"message":  "Rename method 'expectMsgPF' to match camel case naming rules, consider using 'ExpectMsgPf'.",
```

Comment:
Haven't read the full description of the rule but is it allowed to have 2 letters acronym only if not at the end? If yes then fine, otherwise we have a FP here.